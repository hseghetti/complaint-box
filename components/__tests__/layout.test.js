import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../layout';

describe('Layout Component', () => {
  it('Should render Layout', () => {
    const component = renderer.create(
      <Layout />
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});