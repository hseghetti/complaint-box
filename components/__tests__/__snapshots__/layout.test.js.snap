// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout Component Should render Layout 1`] = `null`;
